argocd_applications/creating-an-application-using-gitops-argocd-cli.adoc

```diff
@@ -0,0 +1,30 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="creating-an-application-using-gitops-argocd-cli"]
+= Creating an application by using the GitOps CLI
+:context: creating-an-application-using-gitops-argocd-cli
+
+toc::[]
+
+With Argo CD, you can create your applications to the OpenShift cluster either by using the {gitops-shortname} `argocd` CLI.
+
+[id="prerequisites_creating-an-application-using-gitops-argocd-cli"]
+== Prerequisites
+
+* You have installed the {gitops-title} Operator in your cluster.
+* You have logged into Argo CD instance.
```

Comment:
Do we want to mention this as a pre-requisite ? By this step, do you mean users need to execute `argocd login` command ? If the user is running the command in core mode, then login step is not required.

Reply:
@anandf Good point. I have addressed this issue by moving prerequisites inside the modules for creating and syncing an app in both modes and removing the 2nd bullet.

PTAL and provide feedback. TY!